src/javascript/app_2/App/Components/Elements/tabs.jsx

```diff
@@ -0,0 +1,141 @@
+import classNames from 'classnames';
+import React      from 'react';
+import PropTypes  from 'prop-types';
+
+class Tabs extends React.PureComponent {
+    constructor(props) {
```

Comment:
@zakibinary 
There isn't any need to define constructor. you can define the state like `PopConfirm` component.